jobbergate-cli/pyproject.toml

```diff
@@ -20,44 +20,43 @@ classifiers = [
 "Changelog" = "https://github.com/omnivector-solutions/jobbergate/blob/main/jobbergate-cli/CHANGELOG.rst"
 
 [tool.poetry.dependencies]
-python = "^3.6.2"
-click = "^8.0.1"
-inquirer = "^2.7.0"
+python = "^3.8"
+click = "^8.1.0"
+inquirer = "^2.10.1"
 PyYAML = "6.*"
-python-dotenv = "^0.19.0"
-sentry-sdk = "^1.4.3"
+python-dotenv = "^1.0.0"
+sentry-sdk = "^1.29.2"
 loguru = "^0.6.0"
 python-jose = "^3.3.0"
-pydantic = "^1.8.2"
-httpx = "^0.22.0"
+pydantic = "^1.10.12"
+httpx = "^0.24.1"
 pendulum = "^2.1.2"
-py-buzz = "^3.1.0"
+py-buzz = "^3.2.0"
 rich = "^11.2.0"
-typer = "^0.4.0"
+typer = "^0.9.0"
 pyperclip = "^1.8.2"
 importlib-metadata = "^4.2"
-pep562 = "^1.1"
 
 [tool.poetry.dev-dependencies]
-black = "^21.9b0"
-pytest-cov = "^3.0.0"
+black = "^23.0"
+pytest-cov = "^4.0.0"
 pytest-random-order = "^1.0.4"
 pytest-freezegun = "^0.4.2"
 pytest-responsemock = "^1.0.1"
 isort = "^5.9.3"
 pytest-env = "^0.6.2"
-respx = "^0.19.2"
+respx = "^0.20.1"
 plummet = { extras = ["time-machine"], version = "^1.1.0" }
-pyproject-flake8 = "^0.0.1-alpha.2"
-mypy = "^0.931"
+pyproject-flake8 = "^5.0.4"
```

Comment:
We should use flake8-pyproject, because it patches flake8 and you don't have to call a different executable.

Reply:
good suggestion, done in [b1e3d36](https://github.com/omnivector-solutions/jobbergate/pull/304/commits/b1e3d36be1fc5cde94fb0494e50b9025d6ec4689)